cmd/rtrdump/rtrdump.go

```diff
@@ -38,6 +38,8 @@ var (
 	Serial     = flag.Int("serial.value", 0, "Serial number")
 	Session    = flag.Int("session.id", 0, "Session ID")
 
+	FlagVersion = flag.Int("rtr.version", 1, "What RTR version you want to use, Version 2 is draft-ietf-sidrops-8210bis-10")
```

Comment:
probably want to set 2 as the default, after all the client dictates what the actual protocol version is going to be

Reply:
If I don't do this, then I cannot rtrdump Cloudflare's RTR server, it just instantly hangs up.

I suspect there are more RTR servers like this

Reply:
If the server immediately hangs up, print a warning suggesting the operator to try a different RTR version.

`rtrdump` in StayRTR primarily exists to conform the functioning of StayRTR. Some random.

RTR servers in the wild being incompatible without proper error messages can't be helped.

Having `rtrdump` by default be downgraded to earlier protocols because of buggy third-party implementations stifles innovation.